src/2XXXR0_a_motivation_and_plan_for_a_physical_quantities_and_units_library.md

```diff
@@ -566,6 +568,11 @@ by LEWG will help to keep the efforts on track.
 | 2028.2             | C++29 CS finalized                                 | Resolve any outstanding design/wording issues                                                                                                |
 | 2028.3             |                                                    | Resolve NB comments                                                                                                                          |
 | 2029.1             | C++29 DIS finalized                                | Resolve NB comments                                                                                                                          |
+<!-- LWG will no doubt have their hands full with things like Executors.
+But once the C++29 WP is open for merging, it'd be best to get the papers merged ASAP.
+And then have MSVC, Clang, and GCC's stdlib to implement them ASAP.
+That way, we can get early adoption and UX before C++29 ships, in case any issue pops up.
+Or maybe just target a TS, and strive to get those stdlib implementations to implement them. -->
 
```

Comment:
I totally agree with LWG. That is why in the plan I explicitly stated that I want to move the first papers to LWG immediately when C++29 is open for merging, so LEWG, SG6, and SG16 should be done earlier. SG6 and SG16 parts should be quite easy, but we might have some issues persuading LEWG to discuss C++29 features in the C++26 cycle. That is why I gave ourselves some time for potential slipovers or ping-pong and new revisions after feedback in LEWG.

TS is probably not an option, but we can ask LEWG for recommendations on that.